### PR TITLE
tests: expand spread coverage for stage-snaps

### PR DESCRIPTION
**Pull request by cjp256**
Reproduces and requires #3211 to pass.

### CODE REVIEW

File: tests/spread/general/stage_snaps/task.yaml [hunk 1]
```diff
@@ -1,10 +1,8 @@
 summary: stage-snaps
 
-# We currently only have core18 on a stable channel
 systems:
-  - ubuntu-18.04-64
-  - ubuntu-18.04-amd64
-  - ubuntu-18.04-i386
+  - ubuntu-20.04*
+  - ubuntu-18.04*
```
Comment (jhenstridge) [inline tests/spread/general/stage_snaps/task.yaml:5]: Is this sufficient?  Below it is setting up a project with `snapcraft init`, which would presumably create a core18 basedsnap on all systems.  You'd need a core20 based snap to exercise v2 plugins.

Comment (cjp256) [reply tests/spread/general/stage_snaps/task.yaml:5]: Indeed not. A normal pattern for us is to use `set_base` to correctly set the base to match the host. Added, thanks :)

File: tests/spread/general/stage_snaps/task.yaml [hunk 2]
```diff
@@ -1,20 +1,22 @@
 summary: stage-snaps
 
-# We currently only have core18 on a stable channel
 systems:
-  - ubuntu-18.04-64
-  - ubuntu-18.04-amd64
-  - ubuntu-18.04-i386
+  - ubuntu-20.04*
```
Comment (sergiusens) [inline tests/spread/general/stage_snaps/task.yaml:4]: the architeture specific part of systems can be dropped as the used snaps are no available for all platforms under test. I am weary of this test making our arm64 suite a lot slower than it currently is and have some doubts from the benefits of running these fixtures there, but willing to give it a shot.

**Comment by sergiusens** [comment]
I am closing this in favor of just landing #3211

**Pull request closed without merging**